src/reducers/vis-state-updaters.js

```diff
@@ -1408,9 +1515,13 @@ export function updateAllLayerDomainData(state, dataId, newFilter) {
     }
   });
 
-  return {
+  const newState = {
     ...state,
     layers: newLayers,
     layerData: newLayerDatas
   };
+
+  // return newState;
+  // TODO: H
+  return updateAnimationDomain(newState);
```

Comment:
No need to call it again here
`enableLayerAnimationUpdater` should call `updateAnimationDomain` already